cmake/projects/OpenSSL/schemes/url_sha1_openssl.cmake.in

```diff
@@ -66,13 +66,23 @@ set(configure_command
 # Pass C flags through
 set(configure_opts ${configure_opts} ${CMAKE_C_FLAGS})
 
+# Pass BUILD_SHARED_LIBS (what about RPATH?)
+set(shared_libs "@BUILD_SHARED_LIBS@")
+if (shared_libs)
```

Comment:
> With if(BUILD_SHARED_LIBS), this does not work.

It's not supposed to work like that. BUILD_SHARED_LIBS should affect only current project, if you want to use it for 3rd party, you should use local config. It's possible to introduce global variable for this kind of job, but there is no such feature currently.